### DISCUSSION TITLE
PDF Generator Meeting Output

### DISCUSSION THREAD

**Discussion by Tech4TriCounty** [code4recovery/12-step-meeting-list#1123] [General]
Maybe not for the faint of heart but I would like to be able to edit what pdf generator builds. Is this part of the TSML plugin or ? 
Initially I am only looking to suppress the State as all our meetings are in the same state, so it is superfluous. There may be other things I want to do though I can't think of them. If there is a check box, simple button to this I am all ears. 
![Screenshot 2023-05-02 at 3 37 04 PM](https://user-images.githubusercontent.com/132401769/235780628-14f17bfd-a208-490e-bfd2-b6b398d14642.png)

**Comment by joshreisner** [comment]
this doesn't look like one of our projects - is it https://wordpress.org/plugins/12-step-meeting-pdf-generator/?

**Comment by kiyote33** [comment]
Closing as there has been no response from the submitter.
